<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayOptionsPage</class>
 <widget class="QWidget" name="DisplayOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="GraphTypeGroupBox">
     <property name="title">
      <string>Graphs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label5">
        <property name="text">
         <string>Choose how performance graphs should be displayed:</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="GraphAreasRadioButton">
        <property name="text">
         <string>As filled areas under lines</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pictureBoxArea">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../../resources.qrc">:/icons/grapharea.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="GraphLinesRadioButton">
        <property name="text">
         <string>As lines only</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pictureBoxLine">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../../../resources.qrc">:/icons/graphline.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="TabGroupBox">
     <property name="title">
      <string>Tab Pages</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkBoxStoreTab">
        <property name="text">
         <string>Remember the last selected tab for each object</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="updateLogOptionsDecentGroupBox">
     <property name="title">
      <string>Updates</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="showTimestampsCheckBox">
        <property name="text">
         <string>Show timestamps in Updates log</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
